plugins/jabra-gnp/README.md

```diff
@@ -2,7 +2,8 @@
 
 ## Introduction
 
-This plugin is used to firmware update some Jabra devices.
+This plugin is used to firmware update some Jabra devices,
+notably the Evolve SE (2nd Edition), Evolve2 and Jabra Link connectors.
```

Comment:
I think I would phrase this slightly differently. 
The way I see it we can either:
Start the list of devices with "some" (some Evolve, Evolve2 etc.) and then refer the user to the quirk file, if that's acceptable.
Otherwise list all devices supported. 
The reason is the same stated in the issue, for example we have a Link connector that uses a different protocol and is therefore not supported by this plugin. 
Also, this is missing the Speak2 line

Reply:
@ljrk0 can you fix up the commit and force push your branch to update this PR please.

Reply:
Yep, will do!  Maybe it's actually more helpful if we note by example what lines are not supported and refer to the quirks file?  I've drafted something I'll be pushing now, I does sound more clear from a docs perspective, but I'm interested in your feedback (gosh, writing good and concise docs ain't easy).